File: weeks/03/0_permuting_two_arrays/permuting_two_arrays.rb
def twoArrays(k, a, b)
    a.sort!
    b.sort!.reverse!
  
    a.each_with_index do |_, i|
      if a[i] + b[i] < k
        return 'NO'
      end
    end
  
    return 'YES'
  end
  

File: weeks/03/0_permuting_two_arrays/permuting_two_arrays.js
function twoArrays(k, A, B) {
  A.sort();
  B.sort();
  B.reverse();

  for (let i = 0; i < A.length; i++) {
    if (A[i] + B[i] < k) {
      return "NO";
    }
  }
  return "YES";
}
